### PR TITLE
Release 5.4.2

### PR DESCRIPTION
**Pull request by fredericDelaporte**

### CODE REVIEW

**Review by bahusoid — APPROVED** [review]

File: releasenotes.txt [hunk 1]
```diff
@@ -1,4 +1,27 @@
-﻿Build 5.4.1
+﻿Build 5.4.2
+=============================
+
+Release notes - NHibernate - Version 5.4.2
+
+6 issues were resolved in this release.
```
Comment (hazzik) [inline releasenotes.txt:6]: nit: probably need to mention that it includes fixes from 5.3.16 as well.

Comment (hazzik) [reply releasenotes.txt:6]: I think easiest way would be to create a task 'Merge 5.3.16 into 5.4.x' and mark it as closed by merge commit.

**Comment by bahusoid** [comment]
We should also update link to Nightly builds on nhibernate.info. I think better set it to https://github.com/nhibernate/nhibernate-core#nightly-development-builds

**Comment by fredericDelaporte** [comment]
For nhibernate.info, you mean, posting a news similar to [the MyGet announcement](https://github.com/nhibernate/nhibernate.github.io/blob/master/_posts/2020-05-11-nhibernate-myget-feed-available.md)?

**Comment by bahusoid** [comment]
No. This link:

**Comment by hazzik** [comment]
done

**Review by hazzik — APPROVED** [review]

**Pull request merged**